templates/docker-container-to-acr.yml

```diff
@@ -10,17 +10,17 @@ variables:
   #                          Mandatory variables 
   # ========================================================================
 
-  # Update Azure.ResourceGroupName value with Azure resource group name.
-  Azure.ResourceGroupName: '__TOKEN_AZURE_RESOURCEGROUP_NAME__'
+ # Update Azure.ResourceGroupName value with Azure resource group name.
+  Azure.ResourceGroupName: '{{#toAlphaNumericString repositoryName 50}}{{/toAlphaNumericString}}'
```

Comment:
@ds-ms - I've searched but can't find any documentation about toAlphaNumericString,  shortGuid, etc.  I've never encountered this {{ }} syntax.  Can you provide any link for how to use it?  It seems very cool.

Reply:
Hi Brandon,
I came across this syntax such as toAlphaNumericString, shortGuid and also could not find any documentation. 
Did you find some doc or help in the end?

Thank you,

Kind regards,

Laurent